tests/models/owned_books.js

```diff
@@ -0,0 +1,33 @@
+const ec = require('../../error_codes');
+const initModel = require('../../models/init');
+const ownedBooks = require('../../models/owned_books');
+
+exports.test = function(){
+    /*
+     * Database initialization
+     */
+    initModel.create_tables();
+
+    /*
+     * Test usage of the interface
+     * also shows how to use next callback
+     */
+    ownedBooks.add_book('Adi', 2, test_n);
+    ownedBooks.add_book('Adi', 3, test_n);
+    ownedBooks.get_owners(2, test_next);
+
```

Comment:
Only thing that is scary about this is there is no guarantee that two books are added before get_owners is called.